Doc/library/venv.rst

```diff
@@ -61,7 +61,130 @@ See :pep:`405` for more background on Python virtual environments.
+
+.. highlight:: none
+
+On Windows, invoke the ``venv`` command as follows::
+
+    c:\>Python35\python -m venv c:\path\to\myenv
+
+Alternatively, if you configured the ``PATH`` and ``PATHEXT`` variables for
+your :ref:`Python installation <using-on-windows>`::
+
+    c:\>python -m venv c:\path\to\myenv
+
+The command, if run with ``-h``, will show the available options::
+
+   usage: venv [-h] [--system-site-packages] [--symlinks | --copies] [--clear]
+               [--upgrade] [--without-pip] [--prompt PROMPT] [--upgrade-deps]
+               [--without-scm-ignore-files]
+               ENV_DIR [ENV_DIR ...]
+
+   Creates virtual Python environments in one or more target directories.
+
+   positional arguments:
+     ENV_DIR               A directory to create the environment in.
+
+   options:
+     -h, --help            show this help message and exit
+     --system-site-packages
+                           Give the virtual environment access to the system
+                           site-packages dir.
+     --symlinks            Try to use symlinks rather than copies, when
+                           symlinks are not the default for the platform.
+     --copies              Try to use copies rather than symlinks, even when
+                           symlinks are the default for the platform.
+     --clear               Delete the contents of the environment directory
+                           if it already exists, before environment creation.
+     --upgrade             Upgrade the environment directory to use this
+                           version of Python, assuming Python has been
+                           upgraded in-place.
+     --without-pip         Skips installing or upgrading pip in the virtual
+                           environment (pip is bootstrapped by default)
+     --prompt PROMPT       Provides an alternative prompt prefix for this
+                           environment.
+     --upgrade-deps        Upgrade core dependencies (pip) to the latest
+                           version in PyPI
+     --without-scm-ignore-files
+                           Skips adding SCM ignore files to the environment
+                           directory (Git is supported by default).
+
+   Once an environment has been created, you may wish to activate it, e.g. by
+   sourcing an activate script in its bin directory.
+
+
+.. versionchanged:: 3.4
+   Installs pip by default, added the ``--without-pip``  and ``--copies``
+   options.
+
+.. versionchanged:: 3.4
+   In earlier versions, if the target directory already existed, an error was
+   raised, unless the ``--clear`` or ``--upgrade`` option was provided.
+
+.. versionchanged:: 3.9
+   Add ``--upgrade-deps`` option to upgrade pip + setuptools to the latest on PyPI.
+
+.. versionchanged:: 3.12
+
+   ``setuptools`` is no longer a core venv dependency.
+
+.. versionchanged:: 3.13
+
+   ``--without-scm-ignore-files`` was added along with creating an ignore file
+   for Git by default.
```

Comment:
```suggestion
   Added the ``--without-scm-ignore-files`` option.
   
.. versionchanged:: 3.13
   ``venv`` now creates a :file:`.gitignore` file for the Git VCS by default.
```

Reply:
We've already introduced Git, how about "for Git by default" without needing to refer to it as a VCS?

```suggestion
   Added the ``--without-scm-ignore-files`` option.
   
.. versionchanged:: 3.13
   ``venv`` now creates a :file:`.gitignore` file for Git by default.
```